roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -97,17 +98,28 @@ class Interface(object):
                 self._iftype = match.group(1)
 
     def down(self):
+        if self.ifname.startswith('tap') and self.iftype() == 'tun':
+           return
+
         print "Shutting down ", self
         cmd = self._ip_cmd() + ['link', 'set', 'down', 'dev', self.ifname]
 
         subprocess.check_call(cmd)
 
     def destroy(self):
+        if self.ifname.startswith('tap') and self.iftype() == 'tun':
+           return
+
         print "Deleting interface ", self
         cmd = self._ip_cmd() + ['link', 'delete', 'dev', self.ifname]
-
         subprocess.check_call(cmd)
 
+    def reattach(self):
+        if self._bridge_attached and self.ifname.startswith('tap') and self.iftype() == 'tun':
```

Comment:
the bridge would not exist until after the Neutron  linuxbridge agent as started and completed it's main loop, so we can't reattach it.